### DISCUSSION TITLE
DateTime in Postgresql database

### DISCUSSION THREAD

**Discussion by agwanyaseen** [supabase/supabase#11734] [Questions] [answered]
Hello I was working on supabase c# library for one my side project.
One of the table in database contains created_date, whose value should be datetime of inserted record.
I have been sending this value from client application.
The datatype I have used is Current_TimeStamp.
Now the problem there is a inconsistency of value in my database and in my application.

For eg. I am sending **2023-1-18** from client application, but in the database it is storing **2023-1-17**.
I suspected it is because of DateTimeStyles.AdjustToUniversal (which is by default configured in library).
But not sure if that is the issue or something else. 
What will be the solution to overcome this problem, 
![image](https://user-images.githubusercontent.com/37746694/213169147-ba5a1077-1618-4266-91cc-0bec884c834d.png)

Thank you

**Comment by steve-chavez** [comment, marked as answer]
I suggest asking this on https://github.com/supabase-community/supabase-csharp. The maintainers of the csharp libs are more active on their repos.
